libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/hwdef-bl.dat

```diff
@@ -0,0 +1,109 @@
+# hw definition file for processing by chibios_hwdef.py
+# for the ZeroOneX6 hardware
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# crystal frequency
+OSCILLATOR_HZ 16000000
+
+# board ID for firmware load
+APJ_BOARD_ID 5600
+
+# bootloader is installed at zero offset
+FLASH_RESERVE_START_KB 0
+
+# the location where the bootloader will put the firmware
+FLASH_BOOTLOADER_LOAD_KB 128
+
+# flash size
+FLASH_SIZE_KB 2048
+
+env OPTIMIZE -Os
+
+# order of UARTs (and USB)
+SERIAL_ORDER OTG1 UART7 UART5 USART3
+
+# default to all pins low to avoid ESD issues
+DEFAULTGPIO OUTPUT LOW PULLDOWN
```

Comment:
I think you should just remove this line

Reply:
would this need remain to avoid ESD issues?

Reply:
it should not be used in the bootloader...can be used in main firmware...this prevents some peripherals from entering test modes during their power up